bucket/reaper.json

```diff
@@ -1,6 +1,18 @@
 {
-    "version": "5.980",
     "homepage": "https://www.reaper.fm",
+    "version": "5.980",
+    "license": "Proprietary",
```

Comment:
Is there some URL which could be provided?

Reply:
I haven't found one, only a `license.txt` attached within package.